pkgs/test_core/lib/src/runner/configuration/args.dart

```diff
@@ -124,6 +124,10 @@ final ArgParser _parser = (() {
       defaultsTo: defaultReporter,
       allowed: reporterDescriptions.keys.toList(),
       allowedHelp: reporterDescriptions);
+  parser.addOption('file-reporter',
```

Comment:
Made this a single option to start with, but in the future this could be updated to a multi option to support multiple output formats.